make/common/TestFilesCompilation.gmk

```diff
@@ -89,7 +89,7 @@ define SetupTestFilesCompilationBody
     $$(eval $$(call SetupNativeCompilation, BUILD_TEST_$$(name), \
         NAME := $$(unprefixed_name), \
         TYPE := $$($1_COMPILATION_TYPE), \
-        EXTRA_FILES := $$(file), \
+        EXTRA_FILES := $$(file) $$($1_EXTRA_FILES), \
```

Comment:
Since you are adding a new argument to this function, can you just briefly document it in the list of arguments given at the top of the function (macro) definition?

(Sorry for the slow response; apart from this documentation issue, it looks good).